bigbluebutton-html5/imports/api/slides/server/modifiers/addSlide.js

```diff
@@ -65,8 +65,9 @@ export default function addSlide(meetingId, presentationId, slide) {
 
     const { insertedId } = numChanged;
 
+    requestWhiteboardHistory(meetingId, slide.id);
```

Comment:
We need to request the whiteboard history everytime we upsert the slides, since shapes can be removed while the server is down for example.